src/test/java/core/basesyntax/service/FruitRecordDtoParserFromFileImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.model.FruitRecordDto;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitRecordDtoParserFromFileImplTest {
+    private static FruitRecordDtoParserFromFileImpl fruitRecordDtoParserFromFile;
+    private static final String INCORRECT_STRING = "Mate academy";
+    private static final String CORRECT_STRING = "b,apple,100";
+    private static FruitRecordDto modelFruitRecordDto;
+    private static FruitRecordDto expected;
+    private static FruitRecordDto actual;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        fruitRecordDtoParserFromFile = new FruitRecordDtoParserFromFileImpl();
+        modelFruitRecordDto = new FruitRecordDto();
+        modelFruitRecordDto.setFruit("apple");
+        modelFruitRecordDto.setType(FruitRecordDto.Activities.BALANCE);
+        modelFruitRecordDto.setAmount(100);
+    }
+
+    @Test
+    public void parse_StingIsNull_Ok() {
+        try {
+            fruitRecordDtoParserFromFile.parse(null);
+        } catch (NullPointerException e) {
+            return;
+        }
+        fail("Should throw NullPointerException");
+    }
+
+    @Test
+    public void parse_incorrectString_Ok() {
+        try {
+            fruitRecordDtoParserFromFile.parse(INCORRECT_STRING);
+        } catch (NullPointerException e) {
+            return;
+        }
+        fail("Should throw NullPointerException");
+    }
+
+    @Test
+    public void parse_whatReturns_Ok() {
```

Comment:
```suggestion
    public void parse_validData_Ok() {
```

not informative test method name